<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatOnLineClass</class>
 <widget class="QMainWindow" name="chatOnLineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>99999</width>
    <height>99999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>chatOnLine</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="labelUserInfo">
        <property name="text">
         <string>登陆中……</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="maximumSize">
       <size>
        <width>9999</width>
        <height>9999</height>
       </size>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>180</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>180</number>
      </attribute>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogOut"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogin">
   <property name="text">
    <string>重新登陆</string>
   </property>
  </action>
  <action name="actionLogOut">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="chatonline.qrc"/>
 </resources>
 <connections/>
</ui>
